integrationtest/src/test/java/org/mapstruct/itest/testutil/runner/ProcessorSuiteRunner.java

```diff
@@ -43,11 +40,16 @@
 import org.junit.runner.notification.StoppedByUserException;
 import org.junit.runners.ParentRunner;
 import org.junit.runners.model.InitializationError;
+import org.mapstruct.itest.testutil.runner.ProcessorSuite.CommandLineEnhancer;
 import org.mapstruct.itest.testutil.runner.ProcessorSuite.ProcessorType;
 import org.mapstruct.itest.testutil.runner.ProcessorSuiteRunner.ProcessorTestCase;
 import org.mapstruct.itest.testutil.runner.xml.Toolchains;
 import org.mapstruct.itest.testutil.runner.xml.Toolchains.ProviderDescription;
 
+import static org.apache.maven.it.util.ResourceExtractor.extractResourceToDestination;
```

Comment:
Yeah, sure. It was that kind of change that I got quite often in several files, so I thought you guys have them below the other imports as well - and thus I changed the setting locally. Just changed it back... Should I add a commit to organize all imports again?